README.md

```diff
@@ -10,3 +10,41 @@
 [![Hex.pm](https://img.shields.io/hexpm/dw/chalk_elixir.svg)](https://hex.pm/packages/chalk_elixir)
 
 Elixir Client for Chalk [Online Queries](https://docs.chalk.ai/docs/query-basics).
+
+
+## Usage
+
+Online queries are executed by calling "Chalk.Query.online" and passing inputs and expected outputs as descibed in the [Chalk API documentation](https://docs.chalk.ai/docs/query-basics).
+
+```
+    Chalk.Query.online(%{
+        inputs: %{
+            "user.id": 1
+        },
+        outputs: [
+            "user.id"
+        ]
+    })
+```
+
+## Authentication
+
+The Chalk client uses authentication credentials set in the environment variables `CHALK_CLIENT_ID` and `CHALK_CLIENT_SECRET`.  
+Credentials can be generated on the Chalk dashboard or [via api](https://docs.chalk.ai/docs/online-authentication#fetching-an-access-token).
+
+The credentials can also be passed directly when invoking the client.
+
+```
+    Chalk.Query.online(%{
+        inputs: %{
+            "user.id": 1
+        },
+        outputs: [
+            "user.id"
+        ],
+        context:%{
```

Comment:
I don't think this is right -- I think it's a second arg to the function

Reply:
ur right... my bad, thx